ui_testing/pages/order_page.py

```diff
@@ -343,10 +343,10 @@ def get_suborder_data(self):
             rawTestunitArr = suborder_data['testUnits'].split(',\n')
 
             for testunit in rawTestunitArr:
-                if len(testunit.split(' No: ')) > 1:
+                if len(testunit.split(' Type: ')) > 1:
```

Comment:
Why you update it ?

Reply:
It wasn't working, as `testunit` = str Type: Type name not  str No: No